backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -25,6 +28,29 @@ def get_collection_details(collection_uuid: str):
         raise ForbiddenHTTPException()
 
 
+@db_session
+def create_collection(body: object):
```

Comment:
- `body` is automatically converted to a `dict` by connexions. So you don't need line 35.
- Once you add security in the openapi spec, the `user` param will be magically populated with the owner id.

```suggestion
def create_collection(body: dict, user: str):
```

Reply:
oh awesome, Thanks!